micropip/package_index.py

```diff
@@ -214,6 +214,9 @@ def _fast_check_incompatibility(filename: str) -> bool:
     if not filename.endswith(".whl"):
         return False
 
+    if filename.endswith("wasm32.whl") and sys.platform == "emscripten":
+        return True
```

Comment:
Maybe `sys.platform` check is redundant, as we never expect micropip to be used in the non-Emscripten environment.

Reply:
At least not for installation. For making lock files, it would make sense.

Reply:
> For making lock files

Do you mean using it under `pyodide venv` environment? I think it would still have `sys.platform == emscripten`.

Reply:
Well; at least some test locally runs under `sys.platform != emscripten`, and are useful for fast iteration.